src/zoned.rs

```diff
@@ -413,6 +413,40 @@ impl Zoned {
             .expect("system time is valid")
     }
 
+    /// Returns the current system time with the specific time zone.
```

Comment:
Can you add that this is a convenience function for `Zoned::new(Timestamp::now(), time_zone)`?